src/Adapter/MSTest.TestAdapter/Execution/TestClassInfo.cs

```diff
@@ -21,7 +23,7 @@ namespace Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.Execution;
 /// </summary>
 public class TestClassInfo
```

Comment:
As far as I can see on GitHub and internal repo this is not used outside of MSTest so changes to public API are fine (I will do a follow-up PR changing accessibility where possible to reduce un-needed/wanted public surface).